### ISSUE TITLE
Live2D support

### ISSUE THREAD

**Issue by sorae42** [AppIeWorm/Violet#7]
Is there ever any support for adding Live2D model?

**Comment by MatoiDev** [comment]
Yep, I added support gif, mp4 and live. I'm waiting for the new jb (A12+&iOS15+) to be released in order to test everything and post it right away with support for ios13-15.

**Comment by corruptedkiwi** [comment]
does this mean violet pro can do customs gifs?
